azurelinuxagent/common/protocol/restapi.py

```diff
@@ -282,16 +282,16 @@ def get_ext_handlers(self):
     def get_ext_handler_pkgs(self, extension):
         raise NotImplementedError()
 
-    def get_in_vm_artifacts_profile(self):
+    def get_artifacts_profile(self):
```

Comment:
I think there are just so many similar names which contains artifacts or profile(i.e. extension artifacts) so I am just worried that there might be another thing called 'artifact profile' different from 'in_vm_artifacts_profile'